root.go

```diff
@@ -48,6 +49,7 @@ func init() {
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.app.yaml)")
+	rootCmd.PersistentFlags().StringVar(&substitutionFile, "substitution", "", "parameters substitution file")
```

Comment:
I think that because here we only have config file and not cfgDir we must use a separate file.
In Kube-bench we assume the config.yaml file is in cfgDir which contains master.yaml node.yaml and config.yaml